gradle.properties

```diff
@@ -0,0 +1,3 @@
+android.enableJetifier=true
+android.useAndroidX=true
+org.gradle.jvmargs=-Xmx1536M
```

Comment:
😮

Reply:
Without that file you will see the duplicate classes error below when running `./gradlew ":owncloudComLibrary:assembleAndroidTest" ":owncloudComLibrary:assembleDebug" ":owncloudComLibrary:assembleDebugAndroidTest" ":owncloudComLibrary:assembleDebugUnitTest" ":owncloudComLibrary:assembleRelease" ":owncloudComLibrary:assembleReleaseUnitTest"` command (this is executed on Bitrise)

<img width="840" alt="Screenshot 2020-02-06 at 14 14 50" src="https://user-images.githubusercontent.com/15261363/73940173-4166ec00-48eb-11ea-8f39-3c1b224a4e29.png">